src/ensembl/src/content/app/entity-viewer/shared/components/entity-viewer-sidebar/entity-viewer-sidebar-modal/modal-views/EntityViewerDownloads.tsx

```diff
@@ -24,20 +24,21 @@ import {
 } from 'src/content/app/entity-viewer/state/general/entityViewerGeneralSelectors';
 
 import { parseEnsObjectId } from 'src/shared/state/ens-object/ensObjectHelpers';
+import { isTranscriptProteinCoding } from 'src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary';
```

Comment:
`EntityViewerDownloads` component should not depend on the `GeneSummary` component.